RocketChat/Rocket.Chat
# [FIX][ENTERPRISE] Omnichannel Department form is not correctly storing the list of departments allowed for forwarding

## Proposed changes


The field **List of departments allowed for forwarding** is storing just the first element in the list.
It's necessary to pass the entire array to the server side

Another fix it's to not show the origin department at the list.

## Issue(s)


## How to test or reproduce


On omnichannel panel, go to departments > new/edit department

Try to add many departments at "List of departments allowed for forwarding" field and save.

You'll see that is storing just the first item of the list.

## Screenshots
Many departments
![image](https://user-images.githubusercontent.com/2493803/101403862-95987f00-38b4-11eb-9aca-8217f7bd4543.png)

Department list without origin
![image](https://user-images.githubusercontent.com/2493803/101504158-a814c600-3951-11eb-9c5b-7f45c6b4207d.png)


## Types of changes




## Checklist



## Changelog




## Further comments
